src/odb/src/db/dbBTerm.cpp

```diff
@@ -428,6 +439,32 @@ dbNet* dbBTerm::getNet()
   return nullptr;
 }
 
+dbModNet* dbBTerm::getModNet()
+{
+  _dbBTerm* bterm = (_dbBTerm*) this;
+  if (bterm->_mnet) {
+    _dbBlock* block = (_dbBlock*) getBlock();
+    _dbModNet* net = block->_modnet_tbl->getPtr(bterm->_mnet);
+    return (dbModNet*) net;
+  }
+  return nullptr;
+}
+
+void dbBTerm::connect(dbModNet* mod_net)
```

Comment:
There should be some consistency check to ensure the dbModNet and the dbNet are in sync.  Some with all the other connect methods.